pandas/core/arrays/datetimelike.py

```diff
@@ -127,10 +127,14 @@ def values(self):
     @property
     def asi8(self):
         # do not cache or you'll create a memory leak
-        return self.values.view('i8')
```

Comment:
values doesnt exist on the array classes.  Have I mentioned how much I want to get the arithmetic tests running on these?